README.md

```diff
@@ -50,6 +50,29 @@ You can try out CBL-Mariner with the following ISO: [x86_64 ISO](https://osrelea
 1. *[Optional]* Shut down VM and adjust boot order to place Hard Drive first. Otherwise if Network boot occurs, your VM will attempt to PXE boot with a timeout of about 2 minutes before trying the next boot option.
 1. When prompted sign in to your CBL-Mariner system using the user name and password provisioned through the Installer.
 
+### Verify ISO via Checksum
+We also provide the ISO checksum in a file and GPG signature of that file so you can independently verify the contents.
+
+Steps:
+1. Install `gpg` onto your system.
+   - On Ubuntu, you can run: `sudo apt-get install gpg`
+1. Download Mariner RPM key from the GitHub repo: [Mariner RPM key](https://github.com/microsoft/CBL-Mariner/blob/1.0/SPECS/mariner-repos/MICROSOFT-RPM-GPG-KEY)
+1. Import the Mariner RPM key into gpg: `gpg --import <path to key>`
+1. Download [ISO Checksum](https://osrelease.download.prss.microsoft.com/pr/download/Mariner-1.0-x86_64.iso.sha256) and [ISO Checksum Signature](https://osrelease.download.prss.microsoft.com/pr/download/Mariner-1.0-x86_64.iso.sha256.gpg) files.
+1. Verify the checksum file's signature is associated with this checksum file using gpg: `gpg --verify <path to checksum signature file> <path to checksum file>`
+   - Sample output:
+   ```

```

Comment:
Nit: you could use `'''bash` instead of plain `'''` (they were supposed to be backticks, but don't know how to quote them :P) - this can improve the way this block is rendered a little bit.

Reply:
Fixed in latest